### PR TITLE
feat : mobile responsive

### PR DESCRIPTION
**Pull request by mikesch803**
1. add mobile responsive

## [preview](https://deploy-preview-17--teal-madeleine-ed3325.netlify.app/)

### CODE REVIEW

File: src/components/navbar-bottom/NavbarBottom.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    return {
+      color: isActive ? `` : "black",
+    };
+  };
+  return (
+      <div className="navbar-bottom">
+        <div className="navbar-bottom-icons">
+        {asideData.map((link, index) => (
+          <NavLink style={NavLinkStyles} to={link.to} className={NavLinkStyles?'btn btn-link':'btn'} key={index}>
+            <span className="">{link.icon}</span>
```
Comment (BalajiHarinath) [inline src/components/navbar-bottom/NavbarBottom.jsx]: Suggestion : Can remove the empty classname

File: src/common.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.navbar-bottom {
+  display: none;
+}
+
+@media only screen and (max-width: 945px) {
+  html {
+    font-size: 85%;
+  }
```
Comment (BalajiHarinath) [inline src/common.css]: Suggestion:- Can reduce the font size of the root instead of the entire html since I see you have used rems at many places.

**Review by BalajiHarinath — APPROVED** [review]

File: src/common.css (hunk 2 above)
Comment (mikesch803) [reply src/common.css]: i guess html itself is root, m i right?

File: src/components/navbar-bottom/NavbarBottom.jsx (hunk 1 above)
Comment (mikesch803) [reply src/components/navbar-bottom/NavbarBottom.jsx]: done

File: src/common.css (hunk 2 above)
Comment (BalajiHarinath) [reply src/common.css]: The :root CSS pseudo-class matches the root element of a tree representing the document. In HTML, :root represents the <html> element and is identical to the selector html , except that its specificity is higher.

Comment (BalajiHarinath) [reply src/common.css]: So here yes, it does not matter much.

File: src/components/navbar-bottom/NavbarBottom.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.navbar-bottom {
+  background-color: var(--COLOR-FT);
```
Comment (anik31) [inline src/components/navbar-bottom/NavbarBottom.css:2]: suggestion - you can give more descriptive var names

**Review by anik31 — APPROVED** [review]
The app looks nice in the mobile view. Great work

**Pull request merged**